gpytorch/lazy/kronecker_sum_lazy_tensor.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+
+from .. import settings
+from .kronecker_product_lazy_tensor import KroneckerProductLazyTensor
+from .sum_lazy_tensor import SumLazyTensor
+
+
+class KroneckerSumLazyTensor(SumLazyTensor):
```

Comment:
We may want to consider renaming this? The Kronecker sum has an alternative definition: https://mathworld.wolfram.com/KroneckerSum.html